store/src/app/products/components/footer/footer.component.html

```diff
@@ -0,0 +1,4 @@
+<a class="section-products__link" href="/products">
```

Comment:
Do not see the reason to use here BEM-like class name. It is only one element here, without any nesting. Component - independent part. Do not think about it in the context of parent elements and classes. It should be as dumb as possible

Reply:
Done